alonzo/impl/src/Cardano/Ledger/Alonzo/TxInfo.hs

```diff
@@ -138,33 +142,56 @@ transAddr (AddrBootstrap _bootaddr) = Nothing
 -- ===============================
 -- Translate ValidityIntervals
 
-transVI :: ValidityInterval -> P.SlotRange
-transVI (ValidityInterval SNothing SNothing) =
+transVI ::
+  EpochInfo Identity ->
+  SystemStart ->
+  ValidityInterval ->
+  P.SlotRange
+transVI _ _ (ValidityInterval SNothing SNothing) =
```

Comment:
are the two `_`'s here temporarily, until you can use `transVITime`?

Reply:
yup, I put the EpochInfo and SystemStart parameters here (since we are still calling this function) to see where they propagate to